src/lib/services/index.ts

```diff
@@ -406,7 +407,7 @@ export const createServices = (
 
     const personalDashboardService = new PersonalDashboardService(
         db
-            ? new PersonalDashboardReadModel(db)
+            ? new PersonalDashboardReadModel(db, new ProjectOwnersReadModel(db))
```

Comment:
Since we already have a personal dashboard service maybe it should do the orchestration instead of one read model depending on the other read model?